src/modules/video_player/index.js

```diff
@@ -80,6 +81,18 @@ class VideoPlayerModule {
         watcher.on('load.player', () => {
             this.clickToPause();
             watchPlayerRecommendationVodsAutoplay();
+
+            const currentPlayer = twitch.getCurrentPlayer();
+            this.isMutedByUser =
+                currentPlayer &&
+                currentPlayer.player &&
+                currentPlayer.player.getMuted();
+
+            this.onVisibilityChange = this.onVisibilityChange.bind(this);
```

Comment:
rather than bind to `this` you generally want to define the method as this instead:

```

onWindowBlur = () => ...
```

in reality you shouldn't really need this though.. you should define handle functions outside of this class

Reply:
Yeah, I've updated that, no reason to be class methods.